bin/wp-config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
I'm now also calling it in the prestart.sh script. i was initially just doing a consul template call in there, but it makes more sense to call this script directly and template out the file like that. Also updated the naming of the script